### PR TITLE
[Es] 010 bash conditionals - Add missing EOL

### PR DESCRIPTION
**Pull request by marcelozarate**
## What type of PR is this? (check all applicable)

- [ ] ♻️ Refactor
- [x] ✨ Feature
- [ ] 🐛 Bug Fix
- [ ] 👷 Optimization
- [ ] 📝 Documentation Update
- [ ] 🚩 Other

## Description

Translation of chapter 010 - Bash Conditionals to Spanish

## Added to documentation?

- [ ] 📜 readme
- [x] 🙅 no documentation needed

## [optional] What gif best describes this PR or how it makes you feel?

### CODE REVIEW

File: ebook/es/content/006-bash-comments.md [hunk 1]
```diff
@@ -23,4 +23,4 @@ echo "Hola $nombre"
 echo "¡Bienvenido a DevDojo!"
 ```
 
-Los comentarios son una excelente manera de describir algunas de las funciones más complejas directamente en sus scripts para que otras personas puedan orientarse en su código con facilidad.
\ No newline at end of file
+Los comentarios son una excelente manera de describir algunas de las funciones más complejas directamente en sus scripts para que otras personas puedan orientarse en su código con facilidad.
```
Comment (marcelozarate) [inline ebook/es/content/006-bash-comments.md:26]:
```suggestion
Los comentarios son una excelente manera de describir algunas de las funciones más complejas directamente en sus scripts para que otras personas puedan orientarse en su código con facilidad.

```

File: ebook/es/content/007-bash-arguments.md [hunk 2]
```diff
@@ -74,4 +74,4 @@ echo "El nombre del archivo es: $0 y se va a autoeliminar."
 rm -f $0
 ```
 
-Debe tener cuidado con la eliminación automática y asegurarse de tener una copia de seguridad de su script antes de eliminarlo automáticamente.
\ No newline at end of file
+Debe tener cuidado con la eliminación automática y asegurarse de tener una copia de seguridad de su script antes de eliminarlo automáticamente.
```
Comment (marcelozarate) [inline ebook/es/content/007-bash-arguments.md:77]:
```suggestion
Debe tener cuidado con la eliminación automática y asegurarse de tener una copia de seguridad de su script antes de eliminarlo automáticamente.

```

File: ebook/es/content/009-bash-conditional-expressions.md [hunk 3]
```diff
@@ -184,4 +184,4 @@ Al igual que con otros lenguajes de programación, puede usar las condiciones `A
 
 ```bash
 [[ $? -gt 0 ]]
-```
\ No newline at end of file
+```
```
Comment (marcelozarate) [inline ebook/es/content/009-bash-conditional-expressions.md:187]:
```suggestion
```

```

File: ebook/es/content/010-bash-conditionals.md [hunk 4]
```diff
@@ -0,0 +1,187 @@
...
+
+A continuación, con una sentencia `case`, comprobamos el nombre de la marca y si coincide con alguno de nuestros patrones, imprimimos la ubicación de la fábrica.
+
+Si el nombre de la marca no coincide con ninguna de nuestras sentencias `case`, mostraremos nuestro mensaje por defecto: `es una marca de automóviles desconocida`.
+
+## Conclusión
+
+Le aconsejo intentar modificar el script y jugar con él un poco ¡así puede practicar lo que ha aprendido en los últimos dos capítulos!
+
+Para más ejemplos de sentencias `case` de Bash, asegúrese de ir al capítulo 16, donde crearemos un menú interactivo en Bash usando sentencias `case` para procesar la entrada del usuario.
```
Comment (marcelozarate) [inline ebook/es/content/010-bash-conditionals.md:187]:
```suggestion
Para más ejemplos de sentencias `case` de Bash, asegúrese de ir al capítulo 16, donde crearemos un menú interactivo en Bash usando sentencias `case` para procesar la entrada del usuario.

```

**Review by marcelozarate — COMMENTED** [review]
Somehow Git is removing empty lines at EOF. Suggestions should fix that.

**Review by bobbyiliev — APPROVED** [review]
Thank you!

**Pull request merged**